src/liblsquic/lsquic_engine.c

```diff
@@ -576,6 +576,9 @@ lsquic_engine_new (unsigned flags,
     engine->pub.enp_lookup_cert  = api->ea_lookup_cert;
     engine->pub.enp_cert_lu_ctx  = api->ea_cert_lu_ctx;
     engine->pub.enp_get_ssl_ctx  = api->ea_get_ssl_ctx;
+
+    if (api->ea_generate_cid) engine->pub.enp_generate_cid = api->ea_generate_cid;
```

Comment:
Nit: this can be an unconditional assignment.

Reply:
done